JavaScript/index.js

```diff
@@ -0,0 +1,325 @@
+//Starts Game Cells as Empty
+let board = ["", "", "", "", "", "", "", "", ""];
+
+//Sets Starting Player as Player X
+let currentPlayer = "X";
+
+//Player VS Player Start Button
+let start = document.getElementById("start");
+
+//Player VS Computer Start Button
+let computer = document.getElementById("computer");
+
+//Game Status Text
+let statusText = document.getElementById("status-text");
+
+//Player Names
+let playerNameInput = document.getElementById("player-form");
+
+//Game Timer
+let clock = document.getElementById("clock");
+
+//Empty Player Name Text
+let playerOne = "";
+let playerTwo = "";
+
+//Submit Form Starts as Deactivated
+submit.disabled = true;
+
+//Event Listener placed to allow Players to Enter their Names
+playerNameInput.addEventListener("submit", (evt) => {
+  evt.preventDefault();
+  playerOne = document.getElementById("player-one");
+  playerTwo = document.getElementById("player-two");
+  if (playerOne.value !== undefined) {
+    statusText.textContent = `${playerOne.value}'s Move`;
+  } else {
+    statusText.textContent = `Player X's Move`;
+  }
+});
+
+//Stored Winning Cell Combinations
+let winCombo = [
+  [0, 1, 2],
+  [3, 4, 5],
+  [6, 7, 8],
+  [0, 3, 6],
+  [1, 4, 7],
+  [2, 5, 8],
+  [0, 4, 8],
+  [2, 4, 6],
+];
+
+//Checks Winning Combinations Against Player Cells
+function winCheck() {
+  for (let i = 0; i <= 7; i++) {
```

Comment:
SINGLE LETTER VARIABLE!

Reply:
Also you could use a `for(let arrayItem of winCombo){...}` and it will function the same way and be MUCH more readable